php-cs-fixer

```diff
@@ -23,7 +23,12 @@ if (defined('HHVM_VERSION_ID')) {
     } else {
         exit(1);
     }
-} elseif (!defined('PHP_VERSION_ID') || \PHP_VERSION_ID < 50600 || \PHP_VERSION_ID >= 80100) {
+} elseif (
+    !defined('PHP_VERSION_ID')
+    || \PHP_VERSION_ID === 80000
```

Comment:
This is a bug. Version 8.0.* is supported.

Reply:
What is a bug? Looks fine.

Reply:
@hason , no.
8.0.0 is not supported due to description from few lines below.
8.0.1+ is supported